### ISSUE TITLE
37th online meetup, 2021-07-31

### ISSUE THREAD

**Issue by jongfeel** [ThinkAboutSoftware/OnlineSelfCodingGroup#73]
https://meet.google.com/jyx-mxnq-kpk

### 참여 방법:

Assignees에 자신의 github 계정을 self로 추가
2시간 분량의 할 내용에 대해 댓글 작성 (최소 모임 시작 전까지)
빛의 속도 혹은 (주말, 휴일 포함) 최소 3일 내에 구글 캘린더 등록 메일 확인
모임 시간에 각자 개발 관련된 공부 진행

- 시작: 10시 30분
- 시작 토크: 10시 40분까지
- 공부 진행: 12시 30분 종료
- 마무리 토크: 최대한 12시 40분까지, 조금 오버할 수 있음 (오래 하면 30분 얘기하기도 함)

모임 끝난 후 공부한 내용 정리 & 링크 추가 => 최소 다음 모각코 전까지 확인 가능해야 함.

**주의: 회사일 혹은 마감 기한 임박한 일 처리의 경우는 최대한 자제해 주세요. 주말 아침에 일하면 우울하니까요. ㅜㅜ**

**Comment by mhjoon99** [comment]
한이음에서 지원하는 블렌디드 러닝 강좌를 들을 예정입니다.
-내용 : Android Studio를 활용한 앱 만들기 기초 강좌

**Comment by ellen310** [comment]
DP알고리즘 복습 및 BOJ1915 재도전
https://www.acmicpc.net/problem/1915
https://www.youtube.com/watch?v=FmXZG7D8nS4

**Comment by ShimWooJeong** [comment]
파이썬 웹 크롤링 개념 이해 및 간단 예제 도전

**Comment by jongfeel** [comment]
> 한이음에서 지원하는 블렌디드 러닝 강좌를 들을 예정입니다.
> -내용 : Android Studio를 활용한 앱 만들기 기초 강좌

조금 구체적으로 적어주시면 좋을 것 같아요.
예로 아래 영상 중 어떤 영상을 볼 건지 알려주시면 됩니다.

![image](https://user-images.githubusercontent.com/17442457/127501954-49aa1bb0-43a7-4e24-943c-77e6eb2386c2.png)

**Comment by jongfeel** [comment]
> 파이썬 웹 크롤링 개념 이해 및 간단 예제 도전

우정님도

조금 더 구체적으로 써 주세요. 책을 읽는 건지, 어떤 강의를 듣는 건지에 대한 것도 필요하고
간단한 예제도 어떤 건지 내용이 필요합니다.

2시간 (실질적으로는 1시간 ~ 1시간 30분) 안에 할만한 걸로 적어주시면 좋아요.

**Comment by fora22** [comment]
오랜만에 참여하네요..ㅎㅎ

'핸즈온 머신러닝 with Scikit-Learn & Tensorflow'에서 Softmax 함수, SVM(서포트 벡터 머신)에 대한 내용 읽고 [해당 레포지토리](https://github.com/fora22/Data_Science) .ipynb 파일에 예제 코드 및 개념 정리하기

**Comment by JSY8869** [comment]
오랜만에 돌아왔습니다!!ㅎㅎ

## 목표!
### [인프런 사이트에서 스프링 입문 - 코드로 배우는 스프링 부트, 웹 MVC, DB 접근 기술 강의 듣기](https://www.inflearn.com/course/%EC%8A%A4%ED%94%84%EB%A7%81-%EC%9E%85%EB%AC%B8-%EC%8A%A4%ED%94%84%EB%A7%81%EB%B6%80%ED%8A%B8)
- 스프링 DB 접근기술
    - 스프링 jdbcTemplate(11분)
    - JPA(20분)
    - 스프링 데이터 JPA(15분)
## 시간이 남으면...
### [이것이 취업을 위한 코딩테스트다 with 파이썬 12장 7번 외벽 점검 문제 해결!](https://github.com/fora22/CodingTest/tree/JSY8869/Feature/CodingTest/Chapter12/JSY8869/CHAPTER%2012/%EC%8B%A4%EC%A0%84%2012_7)

**Comment by ErigoLee** [comment]
백준 다익스트라 문제
https://www.acmicpc.net/problem/1261
시간이 남을 시
https://programmers.co.kr/learn/courses/30/lessons/72410
이 문제를 도전합니다.
코드 풀이는 https://github.com/ErigoLee/Algorithm
에 올리겠습니다.

**Comment by Yanghaeun716** [comment]
저는 지금 진행 중인 unity프로젝트의 하나의 프리팹을 완성하는데 시간을 보내려고 합니다!

**Comment by Joseph-Cha** [comment]
### 따배C++ 강의 듣고 내용 정리
- 4.9 ~ 5.1 (총 4강)

**Comment by eunyeong-so** [comment]
# 지난 프로젝트의 Todo 이어서 진행

**Comment by jongfeel** [comment]
### [프로젝트 오일러](https://github.com/jongfeel/ProjectEuler)

28번 문제 Number spiral diagonals 코딩
https://projecteuler.net/problem=28

---

정리한 내용, 나중에 올릴 예정
https://github.com/jongfeel/ProjectEuler/tree/master/Problems/Problem28

**Comment by fora22** [comment]
@ShimWooJeong  팀원 분이 누구셨는지 기억이 잘 안나서 멘션 드립니다..ㅎㅎ

[크롤링 관련한 실행 파일](https://drive.google.com/drive/folders/1qpre8NIyEgTn8qWESsYJYFoFcxaQDcje?usp=sharing)입니다. Google Drive에 저장하신 다음 사용하면 될 것 같습니다. Colab에서 실행한거라 Colab에 대해 모르시면 잠깐 검색하고 실행하시면 될 것 같습니다.

이해를 돕기 위해 몇가지 주석이랑 마크다운 글을 써놓았는데 가독성이 그리 좋지는 못하네요. 양해 부탁드려요.ㅎ.ㅎ

**Comment by ErigoLee** [comment]
문제 풀이
백준 다익스트라 1261번 문제
https://github.com/ErigoLee/Algorithm/tree/1261
프로그래머스 문제
https://github.com/ErigoLee/Algorithm/tree/NewIdSuggest

**Comment by ShimWooJeong** [comment]
@fora22 헉 안 그래도 이 날 모각코에 참여했던 팀원들이 말해줬었는데! 시간이 지나고서도 잊지않고 공유해주셔서 감사합니다 크롤링에 대해서 아직 아무것도 모르는 상태라 저한테 큰 도움이 될 것 같아요 다시 한 번 감사합니다(--)(__)

**Comment by jongfeel** [comment]
@eunyeong-so 

진행한 내용의 링크를 추가해 주세요

**Comment by jongfeel** [comment]
@mhjoon99 

혜준님도 제가 추가로 코멘트 드린 것처럼 강좌 링크 추가해 주세요

**Comment by eunyeong-so** [comment]
> @eunyeong-so
> 
> 진행한 내용의 링크를 추가해 주세요

https://github.com/eunyeong-so/Learn-Anything/pull/2 
위 댓글에도 등록해두었습니다.
이전에 사용하던 프로젝트가 몇개 파일이 날라가게 되어서, 새로 브런치를 파서 
업로드하게 되었습니다. 

풀리의 참고사항에 쓴 내용과 관련하여
IMonster, EnumMonsterClass 를 만들어 몬스터타입을 활용하면 어떨까도 생각중인데, 
종필님의 의견이 궁금합니다 🙌

**Comment by jongfeel** [comment]
@eunyeong-so 

- https://github.com/eunyeong-so/Learn-Anything
- 여기 issue에 추가 코멘트를 달아 보겠습니다.
